forge/src/main/java/org/openjdk/skara/forge/HostedRepository.java

```diff
@@ -208,4 +208,11 @@ default URI reviewUrl(Hash hash) {
     default boolean isSame(HostedRepository other) {
         return name().equals(other.name()) && forge().name().equals(other.forge().name());
     }
+
+    /**
+     * Delete deploy keys which are older than 'age' in this repository
+     * The unit of age is hour
+     * The return value is the count of deleted keys
+     */
+    int deleteDeployKeys(int age);
```

Comment:
Instead of using int for age, please use a [Duration](https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html).

Reply:
Sure